engine.py

```diff
@@ -161,9 +172,12 @@ def _set_project(self):
         self.log_info("Setting Softimage project to '%s'" % proj_path)
 
         try:
-            # test to see if the project has already been set to this path:
+            # test to see if the project has already been set to this path
+            # Application.ActiveProject.Path returns a unicode object. If the path contains
+            # non-ascii characters, the comparison will fail since the str and unicode objects
+            # cannot be compared, so we convert the unicode object to a utf-8 string.
             if (Application.ActiveProject.Path 
-                and os.path.normpath(Application.ActiveProject.Path).lower() == os.path.normpath(proj_path).lower()):
+                and os.path.normpath(Application.ActiveProject.Path).lower().encode("utf-8") == os.path.normpath(proj_path).lower()):
```

Comment:
wonder if there should be a check here - just for safety - so that we only encode the object if it is a unicode object - if it is a string, we skip the encode bit. `encode("utf-8")` does work on string objects, but only if they contain 7 bit ascii. Does that make sense?